README_setup.md

```diff
@@ -0,0 +1,42 @@
+# Setup your virtual environment
+
+## Install Python dependencies
+
+From the project top-level:
+
+Set PYTHONPATH environment variable
+
+    export PYTHONPATH=$(pwd)
+
+Create and activate a new virtual environment:
+
+    python3 -m venv venv
+    . ./venv/bin/activate
+
+Install packages specified in the following requirments files:
+
+    pip install -r requirements.txt
+    pip install -r requirements-build.txt
+
+Most common:
+If the dependency wheel files are available, install the wheel files for leaf-common
+and leaf-server-common:
+
+    pip install leaf-common.whl
+    pip install leaf-server-common.whl
```

Comment:
These examples have tripped at least 3 people up. We should be clear that wheel files need to come from us (at least until we start publishing to pypi).  Also wheel files need to have version and python compatibility information in the file name in order to be valid like:

leaf_common-1.2.20-py3-none-any.whl
leaf_server_common-0.1.17-py3-none-any.whl

Reply:
That can change now: adding these dependencies to `requirements.txt` show work now:
```
git+https://github.com/leaf-ai/leaf-common.git@1.2.20
git+https://github.com/leaf-ai/leaf-server-common.git@0.1.17
```

and the `pip install -r requirements.txt` step will take care of it.

Reply:
I'll update the doc and the requirements files then. Thanks